index.bs

```diff
@@ -2702,8 +2702,11 @@ Determining the W3C Decision</h4>
 	those other conditions must also be re-fulfilled.
 
 	<div class="example">
-		For example, to make [=substantive changes=] to a [=Proposed Recommendations=],
-		the [=technical report=] could be returned to [=Candidate Recommendation=].
+		For example, to make [=substantive changes=] to a [=technical report=]
+		prior to publication as a [=Recommendation=]
+		it would need to go through a new [=Update Request=]
+		and be republished as a new [=Candidate Recommendation=],
+		and to fulfill the criteria for advancement.
```

Comment:
```suggestion
		prior to publication as a [=Recommendation=],
		it would need to go through a new [=Update Request=],
		be republished as a new [=Candidate Recommendation=],
		and fulfill the criteria for advancement.
```